src/config.rs

```diff
@@ -134,10 +134,15 @@ fn parse_contents(contents: String) -> Result<Vec<Hotkey>, Error> {
                 return Err(Error::InvalidConfig(ParseError::MissingCommand(real_line_no)));
             }
 
-            let command = lines[i + 1].trim();
+            let command = lines[i + 1];
+
+            // Error if the command doesn't start with whitespace
+            if !command.starts_with(' ') && !command.starts_with('\t') {
```

Comment:
why not use `||`?
I believe sxhkd doesn't specifically need a tab, any whitespace is a valid indicator for a command.

Reply:
That's a `!() && !()`, so that we allow tab and space.

Reply:
Oops didn't notice the `!` looks good to me.